gradle/trace-files.gradle

```diff
@@ -94,7 +94,7 @@ tasks.register('binRT', TraceFilesTask) {
 //
 // Put here modules following the conventional MODULE/columns.lisp naming scheme
 //
-['bin', 'ec_data', 'euc', 'mmu', 'mxp', 'shf', 'wcp', 'rom', 'romLex', 'trm', 'stp'].each {moduleName ->
```

Comment:
no need to delete the mmu

Reply:
He added it back lower down.

Reply:
Exactly! It was due to reading the `constrants.lisp` file.